mlir/lib/Dialect/SparseTensor/IR/SparseTensorDialect.cpp

```diff
@@ -663,6 +666,17 @@ SparseTensorEncodingAttr::verify(function_ref<InFlightDiagnostic()> emitError,
     return emitError() << "unexpected position bitwidth: " << posWidth;
   if (!acceptBitWidth(crdWidth))
     return emitError() << "unexpected coordinate bitwidth: " << crdWidth;
+  if (auto it = std::find_if(lvlTypes.begin(), lvlTypes.end(), isSingletonDLT);
+      it != std::end(lvlTypes)) {
+    if (it == lvlTypes.begin() ||
+        (!isCompressedDLT(*(it - 1)) && !isLooseCompressedDLT(*(it - 1))))
+      return emitError() << "expected compressed or loose_compressed level "
+                            "before singleton level";
```

Comment:
add `{}` here since the statement spans over multiple lines.

Reply:
Just a nit but the guideline is "Similarly, braces should be used when a single-statement body is complex enough that it becomes difficult to see where the block containing the following statement began", which leave some judgement (it's not automatically "multiple lines" I believe?)

Reply:
I see! Thanks for chiming in, Mehdi. Then in this case, I think it's clear to see where the next block starts, so I will keep it as it is.

Reply:
I *personally* follow the convention that whenever the single statement body spans multiple lines and/or the complex condition statement spans multiple lines, I added the brace. But as Mehdi suggested, it is not a fixed rule.